src/intents.cs

```diff
@@ -9099,5 +10342,2474 @@ interface INVoiceShortcutCenter {
+
+		[NullAllowed, Export ("contacts", ArgumentSemantic.Copy)]
+		INPerson[] Contacts { get; }
+
+		[Export ("recordTypeForRedialing", ArgumentSemantic.Assign)]
+		INCallRecordType RecordTypeForRedialing { get; }
+
+		[Export ("callCapability", ArgumentSemantic.Assign)]
+		INCallCapability CallCapability { get; }
+	}
+
+	[Watch (6,0), NoTV, NoMac, iOS (13,0)]
+	[Protocol]
+	interface INStartCallIntentHandling {
+
+		[Abstract]
+		[Export ("handleStartCall:completion:")]
+		void HandleStartCall (INStartCallIntent intent, Action<INStartCallIntentResponse> completion);
+
+		[Export ("confirmStartCall:completion:")]
+		void Confirm (INStartCallIntent intent, Action<INStartCallIntentResponse> completion);
+
+		[Export ("resolveDestinationTypeForStartCall:withCompletion:")]
+		void ResolveDestinationType (INStartCallIntent intent, Action<INCallDestinationTypeResolutionResult> completion);
+
+		[Export ("resolveContactsForStartCall:withCompletion:")]
+		void ResolveContacts (INStartCallIntent intent, Action<NSArray<INStartCallContactResolutionResult>> completion);
+
+		[Export ("resolveCallCapabilityForStartCall:withCompletion:")]
+		void ResolveCallCapability (INStartCallIntent intent, Action<INStartCallCallCapabilityResolutionResult> completion);
+	}
+
+	[Watch (6,0), NoTV, NoMac, iOS (13,0)]
+	[BaseType (typeof (INIntentResponse))]
+	[DisableDefaultCtor]
+	interface INStartCallIntentResponse {
+
+		[Export ("initWithCode:userActivity:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (INStartCallIntentResponseCode code, [NullAllowed] NSUserActivity userActivity);
+
+		[Export ("code")]
+		INStartCallIntentResponseCode Code { get; }
+	}
+
+	[Watch (6,0), NoTV, NoMac, iOS (13,0)]
+	[BaseType (typeof (INIntentResolutionResult))]
+	[DisableDefaultCtor]
+	interface INTaskPriorityResolutionResult {
+
+		[Static]
+		[Export ("successWithResolvedTaskPriority:")]
+		INTaskPriorityResolutionResult GetSuccess (INTaskPriority resolvedTaskPriority);
+
+		[Static]
+		[Export ("confirmationRequiredWithTaskPriorityToConfirm:")]
+		INTaskPriorityResolutionResult ConfirmationRequired (INTaskPriority taskPriorityToConfirm);
+
+		// Fixes bug 43205. We need to return the inherited type not the base type
+		// because users won't be able to downcast easily
+
+		[New]
+		[Static]
+		[Export ("needsValue")]
+		INTaskPriorityResolutionResult NeedsValue { get; }
+
+		[New]
+		[Static]
+		[Export ("notRequired")]
+		INTaskPriorityResolutionResult NotRequired { get; }
+
+		[New]
+		[Static]
+		[Export ("unsupported")]
+		INTaskPriorityResolutionResult Unsupported { get; }
+
+		[New]
+		[Static]
+		[Export ("unsupportedWithReason:")]
+		INTaskPriorityResolutionResult GetUnsupported (nint reason);
+
+		[New]
+		[Static]
+		[Export ("confirmationRequiredWithItemToConfirm:forReason:")]
+		INTaskPriorityResolutionResult GetConfirmationRequired (NSObject itemToConfirm, nint reason);
+	}
+
+	[Watch (6,0), NoTV, NoMac, iOS (13,0)]
+	[BaseTy
```

Comment:
`resolvedURL` -> `resolvedUrl`

Reply:
GitHub seems to be showing this in the wrong location. Search the diff/code for `resolvedURL` and you'll find the spot.

Reply:
Ahh good catch! Fixed!